frontend/src/app/pages/validation-page/validation-page.component.spec.ts

```diff
@@ -19,7 +25,7 @@ describe('ValidationPageComponent', () => {
         fixture.detectChanges();
     });
 
-    it('should be created', () => {
-        expect(component).toBeTruthy();
-    });
+    // it('should be created', () => {
```

Comment:
Could you explain a little bit why we are commenting this code almost everywhere? 😄

Reply:
These tests should be added along components unit tests.